boards/arm/rpi_pico/rpi_pico.yaml

```diff
@@ -12,4 +12,5 @@ supported:
   - uart
   - gpio
   - i2c
+  - spi
```

Comment:
Add SPI to the "Supported Features" table in `boards/arm/rpi_pico/doc.index.rst` also`

Reply:
Fixed it.